src/main/java/edu/phema/elm_to_omop/repository/OmopRepositoryService.java

```diff
@@ -67,6 +86,21 @@ public Concept getConceptMetadata(String id) throws IOException, ParseException
         return concept;
     }
 
+    public CohortDefinitionDTO createCohortDefinition(CohortDefinitionDTO cohortDefintion) throws OmopRepositoryException {
```

Comment:
This is an example of working with the OHDSI WebAPI without any manual serialization.